WatchFaceKotlin/README.md

```diff
@@ -9,8 +9,9 @@ Demonstrates watch faces using the androidX libraries (Kotlin).
 
 Introduction
 ------------
-The AndroidX watch face libraries allow you to develop a watch face service in Kotlin. However,
-[WFF][1] is strongly recommended in preference to this approach.
+The AndroidX watch face libraries allow you to develop a watch face service in Kotlin.
```

Comment:
We'd want to keep the recommendation to use Watch Face Format, right?

Reply:
Thanks, I think I've fixed this!